src/main/java/cs361/battleships/models/Result.java

```diff
@@ -1,31 +1,47 @@
 package cs361.battleships.models;
 
+
+
+
 public class Result {
 
+	private AtackStatus result;
+	private Square location;
+	private Ship ship;
+
 	public AtackStatus getResult() {
 		//TODO implement
```

Comment:
these TODO comments are confusing. Might as well remove it to prevent confusion. Same goes for the comments further down.